### DISCUSSION TITLE
How do I use this library in nextjs

### DISCUSSION THREAD

**Discussion by michalkvasnicak** [panva/jose#125] [Q&A]
I'm trying to use this library in API powered by nextjs but unfortunately I'm unable to import it.

**Example:**

```ts
import jwtVerify from 'jose/jwt/verify';
```

Just throws an error `Module not found: Can't resolve 'jose/jwt/verify'`

**Versions:**

- Next: 10.0.3
- Node: 12.18.4

**Comment by panva** [comment]
https://github.com/panva/jose#runtime-support-matrix


Node.js LTS ^12.19.0 || ^14.15.0 is required for its module resolution features (Subpath exports). Likewise any tool that is resolving imports must support Subpath exports.

**Comment by michalkvasnicak** [comment]
Updated to v12.20.0, removed node modules and reinstalled. Still the same problem, nextjs is not able to compile SSR code.

**Comment by michalkvasnicak** [comment]
It seems that webpack 4.0 used by nextjs does not support this.

**Comment by michalkvasnicak** [comment]
Using yarn `resolutions` field and setting up to `webpack: 5.10.1` helped.
